src/DwitTech.AccountService.Data/Entities/CodeType.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace DwitTech.AccountService.Core.Enums
+{
+    public enum CodeType
+    {
+        Activation
```

Comment:
I thought enum is basically for selecting one value from the possible set of values. Why does CodeType enum have one value?

Reply:
Well... You can have an Enum that u know will have more values in future. In this case, we will use same code type to describe other types of codes that will be stored in future.